buildSrc/src/main/kotlin/CacheRedirector.kt

```diff
@@ -1,136 +0,0 @@
-/*
- * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
- */
-
-import org.gradle.api.*
-import org.gradle.api.artifacts.dsl.*
-import org.gradle.api.artifacts.repositories.*
-import org.gradle.api.initialization.dsl.*
-import java.net.*
-
-private val cacheRedirectorAvaialble: Boolean = isCacheRedirectorAvailable()
-
-/**
- * Enabled via environment variable, so that it can be reliably accessed from any piece of the build script,
- * including buildSrc within TeamCity CI.
- */
-private val cacheRedirectorEnabled =
-    cacheRedirectorAvaialble && (System.getenv("CACHE_REDIRECTOR_ENABLED")?.toBoolean() == true)
-
```

Comment:
Is this change connected with kotlin 1.6.20?

Reply:
Nope, it's dropped unused code